run_scripts/2_fetch_raw_data.sh

```diff
@@ -29,7 +29,8 @@ PIPELINE_CONFIGURATION_FILE_PATH=$3
 DATA_ROOT=$4
 
 mkdir -p "$DATA_ROOT/Raw Data"
+mkdir -p "$DATA_ROOT/Listening Groups"
```

Comment:
Why do you need to download these to a separate folder?

I can't immediately see the advantages, but happy to discuss/be educated :)

Reply:
Found it cleaner to separate this from the activation and survey raw data files.

Reply:
I think I'm hesitant here because it means you'll need to modify all your other scripts to have yet another argument, so you can copy in the listening groups folder. I think these are, from the pipeline's perspective, still "Raw Data" (or 'input' data, we might like to rename that folder). But I also see the argument against having a flat directory structure. Up to you.

Reply:
I see the need to keep a standard pipeline structure but since this is a project-specific requirement I prefer having a granular structure as it will be easier to search and debug listening group-specific issues if need be.  Thanks.